src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  FC, useEffect, useRef, useState,
+} from 'react';
+import cn from 'classnames';
+import { Todo as TodoTypes } from '../../types/Todo';
+
+type Props = {
+  todo: TodoTypes,
+  deleteTodo: (id: number) => void,
+  Loader: boolean;
```

Comment:
```suggestion
  isLoading: boolean;
```
Boolean variables should have prefix `is` or `has`